packages/docs/content/patterns/utility-actions.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Utility Actions
+intro: Utility actions for a webpage including print, email, RSS and more.
+---
+
+import {
+  DownloadIcon,
+  EmailIcon,
+  LinkIcon,
+  PrintIcon,
+  RssIcon,
+  WhatsNewIcon,
+} from '@cmsgov/design-system';
+
+## Guidance
+
+### Examples
+
+<EmbeddedExample>
+  <div className="ds-l-row">
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <PrintIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        Print
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <EmailIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        Email
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <RssIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        RSS
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <LinkIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        Link
```

Comment:
This might be more of a question for @andream-cmsds or left up to how product teams engage with this pattern. What's the intended UX if someone clicks the "Link" Icon? Can this link be used to convey that clicking will copy the current URL and add it to the user's clipboard? What about linking to another page?

If it's the latter, it may be helpful to change this to a [Link](https://design.cms.gov/foundation/typography/links/?theme=core)

Reply:
Yes, by clicking the "Link" utility action, the current URL will be copied and add to the user's clipboard

Reply:
I think the pattern I've seen for this elsewhere is:
1. A <Button> is used in the markup
2. The button triggers a popover to give end users a choice of what kind of link to share (for Twitter, Facebook, or just the url itself
3. Choosing an action in the popover closes it

Without that functionality, I wonder how useful this pattern actually is - especially when browsers have the ability to share links or users can just copy the address in the url bar. This makes me think the teams using this icon probably have some sort of copy to clipboard mechanic in place.